packages/enhanced_gradients/README.md

```diff
@@ -0,0 +1,49 @@
+# enhanced_gradients
+
+This package adds an easy way to enhance your Flutter gradients and color transitions by interpolating colors
+in the HCT color system provided by the [material_color_utilities package](https://pub.dev/packages/material_color_utilities).
+
+## Usage
+
+The package exposes two ways to modify the builtin `LinearGradient`, `RadialGradient` and `SweepGradient` gradients:
```

Comment:
```suggestion
The package exposes two ways to modify the built-in `LinearGradient`, `RadialGradient` and `SweepGradient` gradients:
```

Reply:
:thonk: is the non-separated version really incorrect? I'm ok with either, so sure let's change it to built-in